.github/workflows/dependency_review.yml

```diff
@@ -0,0 +1,22 @@
+name: dependency_review
+
+on:
+  pull_request:
+    branches:
+      - main
+      - hotfix
+      - feature/**
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # version 3.6.0
+      - name: Dependency Review
+        uses: actions/dependency-review-action@v4
```

Comment:
why don't we use the commit hash here?

Reply:
I don't have a strong opinion, we can just like what we do with `checkout`. Seems like only flutter does this and does not use a commit hash https://github.com/search?q=org%3Aaws-amplify%20dependency-review-action&type=code